Shoko.Server/API/v3/Models/Shoko/Episode.cs

```diff
@@ -251,4 +304,117 @@ public class EpisodeIDs : IDs
             #endregion
         }
     }
+
+    public interface IEpisodeMetadata
```

Comment:
Personally, I think this should be split out into its own file.
Multiple classes/interfaces in one class are messy to me and can get confusing.

Reply:
Removed the interfaces